src/amo/reducers/collections.js

```diff
@@ -260,6 +260,12 @@ export type ExternalCollectionDetail = {|
   uuid: string,
 |};
 
+export type ExternalCollectionDetailWithLocalizedStrings = {|
+  ...ExternalCollectionDetail,
+  description: {[lang: string]: string} | null,
```

Comment:
maybe you want to create a type for this map if we use it in many places.

Reply:
Yes, in fact we already have one, and it's even used in the same file! `LocalizedString` - duh.